datafiles/static/hackage.css

```diff
@@ -192,24 +202,68 @@ pre {
 .def { font-weight: bold; }
 
 
+/* @end */
+
+
+/* @group responsive */
+
+@media only screen and (min-width: 950px) {
+  #page-header {
+    text-align: left;
+    text-align: left;
+  }
+
+  #page-header ul.links {
+    list-style: none;
+    text-align: left;
+    float: right;
+    display: inline-table;
+    margin: 2px 2em 0 2em;
+  }
+
+  #content {
+    width: 55vw;
```

Comment:
as this is narrow page, should that be closer to `100vw`?

Reply:
Yes, close to, to keep a minimally acceptable margin.